src/dotnet/Chat.UI.Blazor/Components/ChatView/Items/Attachment/VisualMediaAttachmentLine.razor

```diff
@@ -0,0 +1,53 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+
+@{
+    var lineHeight = "";
+    var itemCls = "image-extra-large";
+    var attachmentCls = "";
+    if (ImageSize == ImageSize.Small) {
```

Comment:
as far as i understand, all this long ifelse statement is about lineHeight calculation. What if we extract local function `GetLineHeight`?